### PR TITLE
codequery 0.18.1

### PR DESCRIPTION
**Pull request by ruben2020**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --new-formula <formula>` (after doing `brew install <formula>`)?

---

### CODE REVIEW

**Comment by zmwangx** [comment]
This won't work on Sierra due to Qt5 dep.

**Comment by DomT4** [comment]
Going to work on Qt5 tonight, with a mallet if necessary.

**Comment by DomT4** [comment]
Made some progress, but will need some more love later today.

File: Formula/codequery.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 class Codequery < Formula
-  desc "Index, query, or search C, C++, Java, Python, Ruby, or Go code"
+  desc "Index, query, or search C, C++, Java, Python, Ruby, Go and Javascript code"
   homepage "https://github.com/ruben2020/codequery"
-  url "https://github.com/ruben2020/codequery/archive/v0.16.0.tar.gz"
-  sha256 "4896435a8aa35dbdca43cba769aece9731f647ac9422a92c3209c2955d2e7101"
-  revision 2
+  url "https://github.com/ruben2020/codequery/archive/v0.18.0.tar.gz"
+  sha256 "1beacde7c84059c1a62d3ccf6bc7fa9c11336f9df95edc358e7e09eceec0fc42"
+  revision 1
```
Comment (ilovezfs) [inline Formula/codequery.rb]: counting starts from 0 for revisions but we don't write `revision 0`, so the line should just be deleted.

Comment (ruben2020) [inline Formula/codequery.rb]: Done. Thanks!

Comment (ilovezfs) [inline Formula/codequery.rb]: Thanks ruben

**Comment by MikeMcQuaid** [comment]
Fails to install: https://bot.brew.sh/job/Homebrew%20Core/9198/version=any_sierra/testReport/junit/brew-test-bot/sierra/install_codequery/

**Comment by MikeMcQuaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**